BinaryObjectScanner/Protection/CDDVDCops.cs

```diff
@@ -104,13 +82,39 @@ public class CDDVDCops : IContentCheck, IExecutableCheck<NewExecutable>, IExecut
             // TODO: Figure out what NE section this lives in
             var neMatchSets = new List<ContentMatchSet>
             {
+                // Checking for variants with one or two spaces, just in case; the Brockhaus DVDs only had one
+                // CD-Cops, ver. 
+                new(new byte?[]
+                {
+                    0x43, 0x44, 0x2D, 0x43, 0x6F, 0x70, 0x73, 0x2C,
+                    0x20, 0x76, 0x65, 0x72, 0x2E, 0x20
+                }, GetVersion, "CD-Cops (Unconfirmed - Please report to us on Github)"),
```

Comment:
If you found it in something, it's no longer unconfirmed. This applies to all of the checks.